<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q3DDebugWidget</class>
 <widget class="QWidget" name="Q3DDebugWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>452</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>250</width>
        <height>792</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="chkShowTileInfo">
           <property name="text">
            <string>Show map tile info</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkShowBoundingBoxes">
           <property name="text">
            <string>Show bounding boxes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkShowCameraViewCenter">
           <property name="text">
            <string>Show camera's view center</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkShowCameraRotationCenter">
           <property name="text">
            <string>Show camera's rotation center</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkShowLightSourceOrigins">
           <property name="text">
            <string>Show light sources</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkStopUpdates">
           <property name="text">
            <string>Stop scene updates</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkDebugOverlay">
           <property name="visible">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Show debug overlay</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="mDebugShadowMapGroupBox">
         <property name="title">
          <string>Debug Shadow Map</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="1">
           <widget class="QComboBox" name="mDebugShadowMapCornerComboBox"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lblDebugShadowMapCorner">
            <property name="text">
             <string>Corner</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblDebugShadowMapSize">
            <property name="text">
             <string>Size</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QgsDoubleSpinBox" name="mDebugShadowMapSizeSpinBox">
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mDebugDepthMapGroupBox">
         <property name="title">
          <string>Debug Depth Map</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="1">
           <widget class="QComboBox" name="mDebugDepthMapCornerComboBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblDebugDepthMapSize">
            <property name="text">
             <string>Size</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lblDebugDepthMapCorner">
            <property name="text">
             <string>Corner</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QgsDoubleSpinBox" name="mDebugDepthMapSizeSpinBox">
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mCameraInfoGroupBox">
         <property name="title">
          <string>Show camera info</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="12" column="1">
           <widget class="QDoubleSpinBox" name="mCameraZ">
            <property name="decimals">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item row="13" column="0">
           <widget class="QLabel" name="lblLookingX">
            <property name="text">
             <string>Looking at X</string>
            </property>
           </widget>
          </item>
          <item row="14" column="0">
           <widget class="QLabel" name="lblLookingY">
            <property name="text">
             <string>Looking at Y</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QDoubleSpinBox" name="mNearPlane">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="decimals">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QDoubleSpinBox" name="mFarPlane">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="decimals">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="lblCameraX">
            <property name="text">
             <string>Camera X pos</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="lblCameraY">
            <property name="text">
             <string>Camera Y pos</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QDoubleSpinBox" name="mCameraX">
            <property name="decimals">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="lblFarPlane">
            <property name="text">
             <string>Far plane</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="lblCameraZ">
            <property name="text">
             <string>Camera Z pos</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="lblNearPlane">
            <property name="text">
             <string>Near plane</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QDoubleSpinBox" name="mCameraY">
            <property name="decimals">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item row="15" column="0">
           <widget class="QLabel" name="lblLookingZ">
            <property name="text">
             <string>Looking at Z</string>
            </property>
           </widget>
          </item>
          <item row="13" column="1">
           <widget class="QDoubleSpinBox" name="mLookingX">
            <property name="decimals">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item row="14" column="1">
           <widget class="QDoubleSpinBox" name="mLookingY">
            <property name="decimals">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item row="15" column="1">
           <widget class="QDoubleSpinBox" name="mLookingZ">
            <property name="decimals">
             <number>6</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
